manifests/fastcgi/server.pp

```diff
@@ -10,6 +10,14 @@
 
   Apache::Mod['fastcgi'] -> Apache::Fastcgi::Server[$title]
 
+  if is_absolute_path($host) {
+    $is_host = false
```

Comment:
Or even ditch the whole $is_whatever and go for:
```puppet
if is_absolute_path($host) {
  $socket = $host
} else {
  $split = split($host, ':')
  validate_ip_address($split[0])
  $_host = $host
}
```
Then
```erb
<%= if @_host then " -host" end %><%= if @socket then "-socket" end %>
```
which leaves room for other types. Are there types other than host or socket?